mne_bids/tests/test_write.py

```diff
@@ -465,6 +465,9 @@ def test_ctf(_bids_validate):
         output_path = _test_anonymize(raw, bids_basename)
     _bids_validate(output_path)
 
+    with pytest.raises(ValueError, match='All measurement dates are None'):
+        get_anonymization_daysback([raw])
```

Comment:
excellent! :)

the real fun in programming is to check all the corner cases and using as simple a logic to make it work